src/test/java/lotto/common/utill/InputParserTest.java

```diff
@@ -0,0 +1,42 @@
+package lotto.common.utill;
+
+import static lotto.common.utill.InputParser.*;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class InputParserTest {
+
+    @DisplayName("String 입력값을 Int로 변환 가능하다.")
+    @Test
+    void testParseToInt() {
+        String str = "1000";
+        assertThat(parseToInt(str)).isEqualTo(1000);
+    }
+
+    @DisplayName("문자를 포함할 때, 예외가 발생한다.")
```

Comment:
정상케이스와 예외케이스 모두 테스트를 잘 추가해주셨네요 👍🏻

DisplayName에 어떤 기능에 대한 테스트인지 적어주시면 다른 사람이 어떤 테스트인지 파악하기 쉬울 것 같아요! 
아래는 예시일 뿐이에요 ㅎㅎ
```suggestion
    @DisplayName("String 입력값을 Int로 변환할 때, 문자를 포함할 경우, 예외가 발생한다.")
```

그리고 parseToList 에 대한 테스트도 추가해보는건 어떨까요? 😀

Reply:
다시 보니 "문자를 포함할 때, 예외가 발생한다."라는 이름은 정확히 어떤 상황인지 한번에 이해하기 어렵네요.
좀 더 문제 상황을 특정하기 용이한 이름으로 지칭할 필요성을 배우게 되네요..

parseToList에 대한 테스트도 추가했습니다!